### DISCUSSION TITLE
README typo?

### DISCUSSION THREAD

**Discussion by peterhpchen** [WhoIsSethDaniel/mason-tool-installer.nvim#13] [General]
I found the [section Events of README](https://github.com/WhoIsSethDaniel/mason-tool-installer.nvim#events) is wrote `MasonToolsUpdateComplete`, but it is `MasonToolsUpdateCompleted` in the [init.lua](https://github.com/WhoIsSethDaniel/mason-tool-installer.nvim/blob/b84f8afd0326a32bea50f8666e7de71df236f7a9/lua/mason-tool-installer/init.lua#L50).

Is it typo?

Thank you.

**Comment by WhoIsSethDaniel** [comment]
Yes. It's a typo.

**Reply by WhoIsSethDaniel** [reply]
I've committed a fix.
